test-series/4/tools/prepare_abscal_model_files.pbs

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#PBS -N validation_abscal_model_prep
+#PBS -q hera
+#PBS -l nodes=1:ppn=1
+#PBS -l walltime=48:00:00
+#PBS -l vmem=256g
+#PBS -j oe
+#PBS -o /lustre/aoc/projects/hera/Validation/test-4.0.0/logs/abscal_file_prep.out
+#PBS -m be
+#PBS -M r.pascua+nrao@berkeley.edu
+
+# This is intended to be run on the heramgr account.
+date
+source ~/.bashrc
+conda activate validation
+
+# Choose one of the days to use as a reference for antenna adjustment.
+# This is a little roundabout, but it works.
+declare obsfiles=()
+for f in $(ls /lustre/aoc/projects/hera/H1C_IDR2/IDR2_2/2458098/*.HH.uvh5); do
+    obsfiles+=($f)
+done
+declare obsfile=${obsfiles[0]}
+
+# Finish setup.
+declare testdir=/lustre/aoc/projects/hera/Validation/test-4.0.0
+declare simfile=$testdir/data/visibilities/sum.uvh5
+declare savedir=$testdir/data/visibilities/abscal_model
+declare Nint_per_file=60
+declare lst_min=0
+declare lst_max=12
```

Comment:
Is this right, should it be 1-7, or do we need a longer range here?

Reply:
The abscal model files are made to extend a bit past the simulated observation bounds. Originally the simulated observations spanned from 1-11 hours LST, so this choice gave us about an hour on each side. If I were to update the parameters, then I would maybe change the upper bound to 8 hours, but it really doesn't matter, up to a relatively small amount of disk space.